src/Angular.js

```diff
@@ -1532,7 +1532,13 @@ function getNgAttribute(element, ngAttr) {
 
 function allowAutoBootstrap(document) {
   var script = document.currentScript;
-  var src = script && script.getAttribute('src');
+
+  // If the `currentScript` property has been clobbered just return false, since this indicates a probable attack
+  if (!(script instanceof window.HTMLScriptElement || script instanceof window.SVGScriptElement)) {
```

Comment:
1. This was the case before anyway (i.e. if `script` was falsy, we returned `true`).
2. If they can clobber `document.currentScript` to be null, I guess we are doomed. DOOOMED 😱